tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Features/Order Summary.feature

```diff
@@ -166,35 +166,65 @@ Scenario: 7. Get the order summary after the user has viewed the Catalogue Solut
         | commencement-date   | incomplete |       |
         | associated-services | incomplete |       |
         | service-recipients  | incomplete | 0     |
-        | catalogue-solutions | complete   |       |
+        | catalogue-solutions | complete   | 0     |
+        | additional-services | incomplete |       |
+        | funding-source      | incomplete |       |
+
+@5123
+Scenario: 8. Get the order summary that includes a list of Catalogue Solutions
+    Given Orders exist
+        | OrderId    | Description   | OrganisationId                       |  CatalogueSolutionsViewed |
+        | C000016-01 | A Description | 4af62b99-638c-4247-875e-965239cd0c48 |  true                     |
+    And Service Recipients exist
+        | OdsCode | Name                      | OrderId    |
+        | Ods1    | Updated Service Recipient | C000016-01 |
+        | Ods2    | Another Name              | C000016-01 |
+    And Order items exist
+        | OrderId    | OdsCode | CatalogueItemName | CatalogueItemType |
+        | C000016-01 | Ods1    | Order Item 1      | Solution          |
+        | C000016-01 | Ods2    | Order Item 2      | Solution          |
+    When the user makes a request to retrieve the order summary with the ID C000016-01
+    Then a response with status code 200 is returned
+    And the order summary is returned with the following values
+        | OrderId    | OrganisationId                       | Description   |
+        | C000016-01 | 4af62b99-638c-4247-875e-965239cd0c48 | A Description |
+    And the order Summary Sections have the following values
+        | Id                  | Status     | Count |
+        | description         | complete   |       |
+        | ordering-party      | incomplete |       |
+        | supplier            | incomplete |       |
+        | commencement-date   | incomplete |       |
+        | associated-services | incomplete |       |
+        | service-recipients  | incomplete | 2     |

```

Comment:
The line endings in this file has changed from CRLF to mixed, can you change it back please